lms/validation/_base.py

```diff
@@ -136,14 +137,18 @@ def parse(self, *args, **kwargs):
         either return the successfully parsed params or raise a validation
         error.
 
-        :raise lms.validation.ValidationError: if the response isn't valid
+        :raise ExternalRequestError: if the response isn't valid
         """
         try:
-            result = self.load(self.context["response"], *args, **kwargs)
+            return self.load(self.context["response"], *args, **kwargs)
```

Comment:
Took the opportunity to stick a `return` here rather than the `return result` that was previously at the bottom of this method (after the whole `try ... except`). Saves a couple of lines of code